pkg/monitortestlibrary/pathologicaleventlibrary/duplicated_event_patterns.go

```diff
@@ -575,7 +575,6 @@ var AllowBackOffRestartingFailedContainer = &SimplePathologicalEventMatcher{
 var EtcdRequiredResourcesMissing = &SimplePathologicalEventMatcher{
 	name:               "EtcdRequiredResourcesMissing",
 	messageReasonRegex: regexp.MustCompile(`^RequiredInstallerResourcesMissing$`),
```

Comment:
I think we can remove all etcd exceptions now, because I've changed the event sink globally for the whole operator

Reply:
at least we can try, guess we can run another payload test to verify

Reply:
This actually doesn't appear to be an exception that allowed more events, it looks like it was just to isolate them to a specific test so it would be easy to trace down how often it's happening. Once we merge this we can look CI fleet wide to find runs where it would break 20, or where it got close. (the flaked tests are over 10)